great_expectations/data_asset/data_asset.py

```diff
@@ -977,6 +977,21 @@ def validate(
 
             expectation_suite_name = expectation_suite.expectation_suite_name
 
+            expectation_meta = copy.deepcopy(expectation_suite.meta)
+            expectation_meta.pop("citations", None)
+            expectation_meta.pop("great_expectations_version", None)
+            expectation_meta.pop("great_expectations.__version__", None)
```

Comment:
Why do you remove these from the `expectation_meta` dict? It might be useful to know if one version of Great Expectations was used to create the suite, and a different version was used to validate the suite.

Reply:
Thinking of it now  Great Expectations Version (suite) will be and on where it validated will be useful in the future.
I Will keep the version but will still remove the citations as they batch_kwargs are already present

Reply:
Hi @sushrut9898 - I would still think that `citations` might be useful, as the batch_kwargs used to get the batch to create the suite might differ from the batch_kwargs used to get the batch to validate the suite.

Reply:
Sounds good